### DISCUSSION TITLE
Nextjs redux integration example considering server side store

### DISCUSSION THREAD

**Discussion by Besuf** [vercel/next.js#26218] [Ideas]
### Describe the feature you'd like to request

Hello can you provide or update your with-redux-toolkit nextJs integration to consider server side store as well. Nothing is mentioned or shown on how we can dispatch action on the server and match it with the client side store

### Describe the solution you'd like

A way we can access the dispatch and state inside context when we use getServerSideProps or getStaticProps

### Describe alternatives you've considered

I have seen integrations in other boilerplates without redux-toolkit but they look verbose and complicated with hydration actions and stuff. I need a general guide line for the recommended way of doing things from the nextJs team to avoid any unnecessary code complications.

**Comment by Besuf** [comment]
@timneutkens any suggestions?

**Reply by MrJmpl3** [reply]
Maybe this can help? https://www.quintessential.gr/blog/development/how-to-integrate-redux-with-next-js-and-ssr

The problem with this example is the types.
